content/docs/start/experiments.md

```diff
@@ -46,10 +46,17 @@ The biggest difference to previous stages in our pipeline is in two new
 sections: `metrics` and `plots`. These are used to mark certain files containing
 experiment "telemetry". Metrics files contain simple numeric values (e.g. `AUC`)
 and plots files contain matrices and data series (e.g. `ROC` or model loss
-plots) that meant to be visualizing and compared.
+plots) that are meant to be visualizing and compared.
 
-> `cache: false` means that those file are small enough and versioned directly
-> with Git.
+The `-M` flag specifies a metrics file produced by this stage that is not
+tracked by DVC. In this case, we can version the file using Git and so we choose
+to ignore it in DVC.  
+The `--plots-no-cache` flag specifies a plot metrics file produced by this
+stage, which is not cached by DVC. If you want DVC to track the metrics and the
+plot metrics file produced, use `-m` and `--plots` instead respectively.
+
+> With cache: false, DVC skips caching the output as we want "scores.json" to be
```

Comment:
`cache: false` (inline code)

`scores.json` and `prc.json` (inline code, mention both files)